tidb-cloud/tidb-cloud-tls-connect-to-dedicated.md

```diff
@@ -28,16 +28,17 @@ In the [TiDB Cloud console](https://tidbcloud.com/), you can get examples of dif
 
 3. On the **Standard Connection** tab of this dialog, follow the three steps to set up the TLS connection.
    - Step 1：Create traffic filter
-   - Step 2：Download TiDB cluster CA
+   - Step 2：Download CA Cert
    - Step 3：Connect with an SQL client
 
 4. Under **Step 1: Create traffic filter** in the dialog, configure the IP addresses that are allowed to access your cluster. For more information, see [Configure an IP access list in standard connection](/tidb-cloud/configure-ip-access-list.md#configure-an-ip-access-list-in-standard-connection).
 
-5. Under **Step 2: Download TiDB cluster CA**, click **Download TiDB cluster CA** to download it locally for client TLS configuration. The TiDB cluster CA ensures that the TLS connection is secure and reliable.
+5. Under **Step 2: Download CA Cert**, click **Download CA Cert** to download it locally for client TLS configuration. The CA Cert ensures that the TLS connection is secure and reliable.
```

Comment:
```suggestion
5. Under **Step 2: Download CA Cert**, click **Download CA Cert** to download it locally for client TLS configuration. The CA certificate ensures that the TLS connection is secure and reliable.
```

Reply:
CA certificate and CA Cert both mean the same thing. Our competing product is also described as CA Cert. There is no need to modify it here.